website/content/en/docs/dev/drivers.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Virtual Machine Drivers
+weight: 15
+---
+
+Lima supports two types of drivers: **internal** and **external**. This architecture allows for extensibility and platform-specific implementations. Drivers are unware whether they are internal or external.
+
+## Internal vs External Drivers
+
+**Internal Drivers** are compiled directly into the `limactl` binary and are registered automatically at startup by passing the driver object into `registry.Register()` function and importing the package in the main limactl code using Go's blank import `_`. For example:
+- **qemu:** [registration file](https://github.com/lima-vm/lima/blob/master/pkg/driver/qemu/register.go) & [import file](https://github.com/lima-vm/lima/blob/master/cmd/limactl/main_qemu.go)
+
+Build tags control which drivers are compiled as internal vs external (e.g., `external_qemu`, `external_vz`, `external_wsl2`).
+
+**External Drivers** are separate executables that communicate with Lima via gRPC. They are discovered at runtime from configured directories.
+
+> **⚠️ Note**: External drivers are experimental and the API may change in future releases.
+
+## Building Drivers as External
```

Comment:
Should be moved to , or at least linked from, https://lima-vm.io/docs/installation/source/